### DISCUSSION TITLE
I cannot download anything

### DISCUSSION THREAD

**Discussion by ghost** [DamianMorozov/OpenTgResearcher#9] [Q&A]
I configured all the settings, they all were green, but after clicking download the program keep showing the message `Thinking` without any download happening. So I don't know what the issue is. You have any suggestions?

**Comment by DamianMorozov** [comment]
Can you attach the screenshot of your "Download settings"?

**Comment by ghost** [comment]
Here it is
![Screenshot](https://user-images.githubusercontent.com/90341705/210395171-3b77d8ed-aafc-438c-817c-00f21d92f85f.png)

**Reply by DamianMorozov** [reply]
Do you have correct "Storage settings" and "Client settings"?
![image](https://user-images.githubusercontent.com/13628284/210407460-dee77f60-eb46-4fa0-87f5-3969f183b868.png)

**Reply by DamianMorozov** [reply]
Can you send me th TG channel ID?

**Reply by ghost** [reply]
Yes, everything is green there.  Check `public_test_ch`

**Reply by DamianMorozov** [reply]
Oh, I see what's the problem.
The first message link `https://t.me/public_test_ch/5878` means start ID is `5878`.
But now, the software starts from ID `1`.
I will thinking how to fix that problem.

**Reply by ghost** [reply]
Great, thanks!

**Reply by DamianMorozov** [reply]
Try the last release v0.1.350.
I fix your problem by setup start message ID.

**Reply by ghost** [reply]
I'm stuck at  `Type the source ID:` I entered the chat id, but the message keeps appearing every time I enter the chat id.

**Reply by DamianMorozov** [reply]
Oh, i see that is.
Fixed it here: [v0.1.360](https://github.com/DamianMorozov/TgDownloader/releases/tag/v0.1.360)
